wifiphisher/macmatcher.py

```diff
@@ -0,0 +1,45 @@
+import os
+from constants import *
+
+class MACMatcher():
+    """
+        This class is using Organizationally Unique Identifiers (OUIs) 
+        in order to match MAC addresses of devices to its vendord  
+
+        See http://standards.ieee.org/faqs/OUI.html
+    """
+    def __init__(self, mac_prefix_file=MAC_PREFIX_FILE):
+        self.mac_to_vendor = {}
+
+        path = os.path.join(os.path.dirname(__file__), '..', mac_prefix_file)
+
+        with open(path, 'r') as prefixes_file:
+            for line in prefixes_file:
+                
+                #skip comments
+                if line[0] == '#':
```

Comment:
instead of doing above I think doing this
```python
if not line.startswith("#"):
    [m, v] = line.rstrip('\n').split(' ', 1)
    self.mac_to_vendor[m] = v
```
is better.